### PR TITLE
[CDAP-13301] Adds metrics for profile and associations for a profile

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-13301
Build: https://builds.cask.co/browse/CDAP-URUT23

### CODE REVIEW

**Comment by ajainarayanan** [comment]
~Working on Profile Preview in pipeline's run level view to add the same metrics. But the existing code can be reviewed independently.~
Right now there is some redundant code that we can refactor for more reuse. Committing this for now.

File: cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/ProfileAssociations/index.js [hunk 1]
```diff
@@ -68,6 +71,83 @@ export default class ProfileAssociations extends Component {
...
+        },
+        "metrics": [
+          "system.program.completed.runs",
+          "system.program.node.minutes"
+        ],
+        "timeRange": {
+          "start": startTime,
+          "end": endTime,
+          "resolution": "auto",
+          "aggregate": true
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/ProfileAssociations/index.js]: do these properties need to be wrapped in quotes?

File: cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/ProfileAssociations/index.js [hunk 2]
```diff
@@ -68,6 +71,83 @@ export default class ProfileAssociations extends Component {
...
+        }
+      }
+    };
+  };
+
+  fetchAggregateMetrics = (startTime, endTime, metadata) => {
+    let metricsBody24 = this.getMetricsQueryBody(startTime, endTime, metadata);
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/ProfileAssociations/index.js]: why 24?

File: cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/index.js [hunk 3]
```diff
@@ -57,6 +66,59 @@ export default class ProfileDetailView extends Component {
...
+        },
+        "metrics": [
+          "system.program.completed.runs",
+          "system.program.node.minutes"
+        ],
+        "timeRange": {
+          "start": startTime,
+          "end": endTime,
+          "resolution": "auto",
+          "aggregate": true
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/index.js]: same here about the quotes

File: cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/index.js [hunk 4]
```diff
@@ -57,6 +66,59 @@ export default class ProfileDetailView extends Component {
...
+        }
+      }
+    };
+  };
+
+  fetchAggregateMetrics = (metricName, startTime, endTime) => {
+    let metricsBody24 = this.getMetricsQueryBody(this.state.profile, startTime, endTime);
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/index.js]: same with 24

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 5]
```diff
@@ -249,6 +249,17 @@ features:
...
+        Associations:
+          Header:
+            created: Created
+            lastrunnodehr: Last run node/hr
+            last24hrsruns: Last 24 hrs runs
+            name: Name
+            namespace: Namespace
+            schedules: Schedules
+            totalruns: Total runs
+            totalnodehr: Total node/hr
```
Comment (elfenheart) [inline cdap-ui/app/cdap/text/text-en.yaml]: flip this with line 260 (alphabetical order)

**Review by elfenheart — APPROVED** [review]
👍 code LGTM

**Review by bdmogal — APPROVED** [review]
As long as there is a jira to track creation date, this lgtm

**Comment by ajainarayanan** [comment]
@bdmogal https://issues.cask.co/browse/CDAP-13764

**Pull request merged**